dolweb/downloads/templates/downloads-index.html

```diff
@@ -31,10 +31,13 @@
 <h1>{% trans "Development versions" %}</h1>
 
 <div class="alert">{% blocktrans %}
-    Development versions are released every time a developer makes a change to
+    <p>Development versions are released every time a developer makes a change to
     Dolphin, several times every day! Using development versions enables you to
     use the latest and greatest improvements to the project. They are however
-    less tested than stable versions of the emulator.
+    less tested than stable versions of the emulator.</p>
+
+    <p>The development versions require 64 bit Visual C++ runtimes for Visual Studio 2015
+    installed: <a href="https://www.microsoft.com/en-us/download/details.aspx?id=48145">Link to download</a></p>
```

Comment:
* 64-bit should be written with a hyphen.
* Is "runtimes" the correct term? I think it looks weird pluralized. The page this links to uses "redistributable" instead.
* Including "Link to" in the link text is unnecessary since it should be obvious that this is a link.